

#by the end ofthe  article you will know what
'''
how to handle exception using the try, except, and finally statement 
'''
from random import gauss


try:
    guss = input("Enter name: ")
    name = bello
    print("my name is koji{name},user {guess}")
except Exception as e:
    print(e)
print("my name is conilius")
    
    
    